.github/workflows/reusable-verify.yml

```
@@ -168,6 +179,16 @@ jobs:
         run: bun run app:build
         env:
           VITE_CONVEX_URL: https://example.convex.cloud
+      - name: Upload bundle stats to Codecov
+        # @codecov/vite-plugin only peers with Vite <=6 (no rolldown support),
+        # so the bundler-agnostic analyzer runs on the finished client build
+        # instead. Not tokenless — skipped on fork PRs where the secret is
+        # absent; never fails the build.
+        if: ${{ secrets.CODECOV_TOKEN != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)